src/test/java/com/wikia/webdriver/common/core/api/CuratedContent.java

```diff
@@ -0,0 +1,55 @@
+package com.wikia.webdriver.common.core.api;
+
+import com.wikia.webdriver.common.core.XMLReader;
+import com.wikia.webdriver.common.core.annotations.User;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.NameValuePair;
+
+import java.io.File;
+import java.net.URL;
+import java.util.List;
+
+public class CuratedContent extends ApiCall {
+
+  private CuratedContent() {
+  }
+
+  @Override protected String getURL() {
+    return "MY AWESOME URL";
+  }
+
+  @Override protected User getUser() {
+     return User.STAFF;
+  }
+
+  @Override protected List<NameValuePair> getParams() {
+    return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1168)